packages/react-vapor/src/components/actionable-item/styles/ActionableItem.scss

```
@@ -6,6 +6,7 @@ $actionable-item-dots-width: 12px;
 
 .actionableItemContainer {
     height: $button-height;
+    background-color: #ffffff; // var(--white);

```

should you use the var instead here?

```suggestion
    background-color: var(--pure-white);

```
```suggestion
    background-color: $pure-white;

```